src/assets/scss/overrides.scss

```diff
@@ -84,3 +75,207 @@
     box-shadow: none !important;
   }
 }
+
+.grey-box-checkbox-button .v-input--selection-controls__ripple {
+  display: none;
+}
+
+.grey-box-checkbox-button .v-input__slot {
+  position: relative;
+  left: 0px;
+  top: 0px;
+}
+
+.designation-buttons button {
+  margin: 0 !important;
+  padding: 0 !important;
+}
+
+.designation-error-col {
+  padding: 5px 10px 0px 10px;
+}
+
+.grey-box-checkbox-button label {
+  font-size: 14px;
+  color: $non-clickable-blue !important;
+}
+
+.v-btn {
+  font-family: inherit !important;
+  font-size: 0.875rem !important;
+  text-transform: none !important;
+  letter-spacing: normal !important;
+  height: 2.5rem !important;
+  box-shadow: 1px 1px rgba(0,0,0,.2) !important;
+  color: white !important;
+  background-color: $app-blue !important;
+  padding: inherit 28px !important;
+}
+
+.v-btn.normal {
+  color: white !important;
+  background-color: $gray6 !important;
+}
+
+.v-label {
+  color: rgba(73, 80, 87, .95) !important;
```

Comment:
Theoretically, this will now affect v-labels that use SCSS (where previously it only affected v-labels that used SASS). Have you noticed any changes to the UI?

Reply:
Since it was imported in main.ts this could have no impact. right?

Reply:
You are right! I confirmed this in NR UI Dev.

![image](https://user-images.githubusercontent.com/10002889/116911311-8c110a00-abfb-11eb-9732-b867eed6e6f8.png)